crates/rerun/src/crash_handler.rs

```diff
@@ -1,55 +1,80 @@
+use re_build_info::BuildInfo;
+
+use parking_lot::Mutex;
+
+// The easiest way to pass this to our signal handler.
+static BUILD_INFO: Mutex<Option<BuildInfo>> = Mutex::new(None);
+
 /// Install handlers for panics and signals (crashes)
 /// that prints helpful messages and sends anonymous analytics.
 ///
 /// NOTE: only install these in binaries!
 /// * First of all, we don't want to compete with other panic/signal handlers.
 /// * Second of all, we don't ever want to include user callstacks in our analytics.
-pub fn install_crash_handlers() {
-    install_panic_hook();
+pub fn install_crash_handlers(build_info: BuildInfo) {
+    install_panic_hook(build_info);
 
     #[cfg(not(target_arch = "wasm32"))]
     #[cfg(not(target_os = "windows"))]
-    install_signal_handler();
+    install_signal_handler(build_info);
 }
 
-fn install_panic_hook() {
+#[cfg(feature = "analytics")]
+fn add_build_info(build_info: &BuildInfo, event: re_analytics::Event) -> re_analytics::Event {
+    event
+        .with_prop("rerun_version", build_info.version)
+        .with_prop("target", build_info.target_triple)
+        .with_prop("git_hash", build_info.git_hash_or_tag())
+        .with_prop("git_branch", build_info.git_branch)
+        .with_prop("build_date", build_info.datetime)
+        .with_prop("debug", cfg!(debug_assertions)) // debug-build?
+        .with_prop(
```

Comment:
It would be nice to be assured that we use the same keys as for other analytics because that might make it easier to filter for things in the future. Can you merge this with the snippet in `viewer_anlyatics.rs`? Btw. the code over there doesn't use `git_hash_or_tag` yet